src/common/ZulipStatusBar.js

```diff
@@ -66,16 +66,12 @@ class ZulipStatusBar extends PureComponent<Props> {
   }
 }
 
-export default connect((state, props) => ({
+export default connect((state: GlobalState, props: Object) => ({
   safeAreaInsets: getSession(state).safeAreaInsets,
   theme: getSettings(state).theme,
-  // Prettier messes up the ternary so the linter complains.
-  // prettier-ignore
   backgroundColor: !props.backgroundColor
```

Comment:
This can be written `props.backgroundColor || getTitleBackgroundColor(props.narrow)(state)` but also we don't want to put that logic here at all (I know you didn't add it in this commit).

Reply:
submitted #2966. :)